packages/frontend-v2/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,35 @@
+import { useState, Dispatch, SetStateAction } from "react";
+
+const isWindow = typeof window !== "undefined";
```

Comment:
why do we need this check in lines 11/25? might be useful to add a comment here explaining why